src/test/java/teammates/test/cases/ui/browsertests/InstructorCommentsPageUiTest.java

```diff
@@ -102,11 +102,13 @@ private void testScripts() throws Exception {
         commentsPage.verifyContains("comments.idOfArchivedCourse");
         
         commentsPage.clickNextCourseLink();
-        assertTrue("URL: " + browser.driver.getCurrentUrl(), browser.driver.getCurrentUrl().contains(Const.ActionURIs.INSTRUCTOR_COMMENTS_PAGE
+        assertTrue("URL: " + browser.driver.getCurrentUrl(),
+                   browser.driver.getCurrentUrl().contains(Const.ActionURIs.INSTRUCTOR_COMMENTS_PAGE
                    + "?user=comments.idOfInstructor1OfCourse1&courseid=comments.idOfArchivedCourse"));
```

Comment:
Might need to indent this 4 spaces to the right now, since it is a part of a parameter and not a parameter on its own.